SSE2NEON.h

```diff
@@ -1194,4 +1194,96 @@ FORCE_INLINE void _mm_clflush(void const*p)
 	// no corollary for Neon?
 }
 
+
+FORCE_INLINE void _mm_pause() {

```

Comment:
I am not if it makes sense to have compiler memory barrier to make sure it waits semantically.